### PR TITLE
Program/Process Command Cleanup and Excel Bug Fixes

### PR DESCRIPTION
**Pull request by openbots-ff**
- Cleaned up Program/Process commands
- Added "Close/Kill" option to Stop Process Command. 

- Added "Close All Existing Excel/Word Instances" option to Create Excel/Word Application Commands.
- Write Range Command now throws an exception when the input for "Cell Location" is a range instead of a single cell.

### CODE REVIEW

File: taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs [hunk 1]
```diff
@@ -1,65 +1,69 @@
...
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Python")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Default")]
-        [Attributes.PropertyAttributes.InputSpecification("Select the type of script you want to execute.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select Powershell to run ps1 files, Default to run a file through the system default.")]
-        [Attributes.PropertyAttributes.Remarks("Default executes with the system default for that file type.")]
+        [PropertyDescription("Script Type")]
+        [PropertyUISelectionOption("Default")]
+        [PropertyUISelectionOption("Powershell")]
+        [PropertyUISelectionOption("Python")]
+        [InputSpecification("Select the type of script you want to execute.")]
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs:35]: Please add the SampleUsage and keep it empty

File: taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs [hunk 2]
```diff
@@ -1,65 +1,69 @@
...
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Enter the path to the script, or select a file.")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a fully qualified path to the script, including the script extension.")]
-        [Attributes.PropertyAttributes.SampleUsage("**C:\\temp\\myscript.vbs**")]
-        [Attributes.PropertyAttributes.Remarks("This command differs from **Start Process** because this command blocks execution until the script has completed.  If you do not want to stop while the script executes, consider using **Start Process** instead.")]
+        [PropertyDescription("Script Path")]
+        [InputSpecification("Enter a fully qualified path to the script, including the script extension.")]
+        [SampleUsage(@"C:\temp\myscript.ps1 || {vScript} || {ProjectPath}\myscript.ps1")]
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs]:
```suggestion
        [SampleUsage(@"C:\temp\myscript.ps1 || {vScriptPath} || {ProjectPath}\myscript.ps1")]
```

**Review by ob-ab — CHANGES_REQUESTED** [review]
Please make a couple of minor changes and everything else looks good to me.

File: taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs (hunk 1 above)
Comment (openbots-ff) [reply taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs:35]: done

**Review by ob-ab — APPROVED** [review]

**Review by CameronHerwig — APPROVED** [review]

**Pull request merged**